codespell_lib/data/dictionary.txt

```diff
@@ -6004,6 +6020,7 @@ foriegn->foreign
 forld->fold
 forlder->folder
 forlders->folders
+forma->format, from,
```

Comment:
Please remove this due to https://en.wikipedia.org/wiki/Pro_forma .

Reply:
ok, removed.